src/einsteinpy/utils/__init__.py

```diff
@@ -1,4 +1,9 @@
 from .bl_coord_transforms import *
 from .coord_transforms import *
 from .scalar_factor import scalar_factor, scalar_factor_derivative
-from .schwarzschild_utils import schwarzschild_radius, time_velocity
+
```

Comment:
remove the newline here, It would be sorted, tests would pass :)

Reply:
> 
> 
> remove the newline here, It would be sorted, tests would pass :)

i still dont understand why isort is not working on my machine.
i did that many times but no use :(

Reply:
`pip uninstall astropy einsteinpy` and then follow the dev version installation instructions http://docs.einsteinpy.org/en/latest/getting_started.html#development-version.

Might help

Reply:
`pip instal --upgrade isort` 
:)